From d8d06ca6ad3cad059e1edf6d0c1ca3c9af12a02f Mon Sep 17 00:00:00 2001
From: Venkateshwar Rao G <venkateshwar.rao.gannavarapu@xilinx.com>
Date: Thu, 8 Feb 2018 15:09:01 -0800
Subject: [PATCH 0261/1851] dt: bindings: display: xlnx: Adding csc bindings

commit 503b522d1746a3160553a1a238098b8c9b4abed9 from
https://github.com/Xilinx/linux-xlnx.git

This patch adds the documentation for xilinx color space converter(csc).

Signed-off-by: Venkateshwar Rao G <venkateshwar.rao.gannavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../bindings/drm/xilinx/xlnx,vpss-csc.txt     | 21 +++++++++++++++++++
 1 file changed, 21 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/drm/xilinx/xlnx,vpss-csc.txt

diff --git a/Documentation/devicetree/bindings/drm/xilinx/xlnx,vpss-csc.txt b/Documentation/devicetree/bindings/drm/xilinx/xlnx,vpss-csc.txt
new file mode 100644
index 000000000000..7b1e29870b96
--- /dev/null
+++ b/Documentation/devicetree/bindings/drm/xilinx/xlnx,vpss-csc.txt
@@ -0,0 +1,21 @@
+Xllinx VPSS Color Space Converter
+-----------------------------------------
+The Xilinx VPSS Color Space Converter is a Video IP that supports
+color space conversion from RGB to YUV 444/422/420 and vice versa.
+
+Required properties:
+
+- compatible: Must be "xlnx,vpss-csc".
+
+- reg: Physical base address and length of registers set for the device.
+
+- xlnx,video-width: This property qualifies the video format with sample
+  width expressed as a number of bits per pixel component. Supported video
+  width values are 8/10/12/16.
+
+Example:
+	csc@a0040000 {
+		compatible = "xlnx,vpss-csc";
+		reg = <0x0 0xa0040000 0x0 0x10000>;
+		xlnx,video-width = <8>;
+	}
-- 
2.31.1

